tests/pipeline/pipeline.go

```diff
@@ -0,0 +1,103 @@
+package pipeline
+
+/* This was generated from a template file. Please feel free to update as necessary */
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/devfile/library/pkg/util"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/redhat-appstudio/e2e-tests/pkg/constants"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils/tekton"
+)
+
+var _ = framework.PipelineSuiteDescribe("Pipeline E2E tests", Label("pipeline"), func() {
+
+	defer GinkgoRecover()
+	var fwk *framework.Framework
+	// use 'f' to access common controllers or the specific service controllers within the framework
```

Comment:
is this comment relevant?

Reply:
@psturc That is a comment from the template to let contributors know they access the needful functions from the framework. Do we want to remove this comment or just change the `f` to `fwk`?

Reply:
> is this comment relevant?
fixed.